scripts/deploy_cf_bosh_instance

```diff
@@ -97,14 +97,12 @@ mkdir -p ${deployment_dir}/generated-stubs/cf
 mkdir -p generated-stubs
 
 # create keypair
-if ! aws ec2 describe-key-pairs --key-names bosh; then
-  aws ec2 create-key-pair --key-name bosh | jq -r .KeyMaterial > ${deployment_dir}/artifacts/keypair/id_rsa_bosh
-fi
+aws ec2 delete-key-pair --key-name bosh
```

Comment:
This script should be idempotent.  If you already have a deployment and an SSH key that you're using, you wouldn't want to have the key changed by running this script.